proposals/NNNN-swiftpm-html-coverage-report.md

```diff
@@ -0,0 +1,185 @@
+# HTML Coverage Report
+
+* Proposal: [SE-NNNN](NNNN-swiftpm-html-coverage-report.md)
+* Authors: [Sam Khouri](https://github.com/bkhouri)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+
+*During the review process, add the following fields as needed:*
+
+* Implementation: [swiftlang/swift-package-manager#9076][PR]
+* Decision Notes: [Pitch](https://forums.swift.org/t/pitch-adding-html-coverage-support/82358)
+
+## Introduction
+
+Currently, `swift test` supports generating a JSON coverage report, which is
+great for ingesting into various systems. The JSON, however, is not very "human
+consumable friendly" while iterating at-desk.
+
+This proposes adding an additional command line argument to `swift test` that
+would allow the caller to select the generation of an HTML coverage report.
+
+
+## Motivation
+
+JSON coverage report is great for ingesting into exgernal tools that post-process
+the coverage data.  Allowing users to generate an HTML coverage report allows
```

Comment:
To make this easier to understand, consider rephrasing it to something structured as "if we do X, then users can do Y." For instance, "If SwiftPM could generate an HTML coverage report, users could upload [...]"